views/user/show.pug

```diff
@@ -4,14 +4,6 @@ block content
   h1.title= userToShow.name
   if user.id === userToShow.id
     h2.subtitle.my-5 Személyes beállítások
```

Comment:
You can remove this line too, as currently there are no settings on the profile page.